File: src/test/java/uk/gov/justice/digital/hmpps/oauth2server/nomis/model/StaffTest.java
package uk.gov.justice.digital.hmpps.oauth2server.nomis.model;

import org.junit.Test;

import static org.assertj.core.api.Assertions.assertThat;

public class StaffTest {

    @Test
    public void getFirstName() {
        final var staff = new Staff();
        staff.setFirstName("boB");
        assertThat(staff.getFirstName()).isEqualTo("Bob");
    }

    @Test
    public void getName() {
        final var staff = new Staff();
        staff.setFirstName("boB");
        staff.setLastName("SMITH");
        assertThat(staff.getName()).isEqualTo("Bob Smith");
    }
}

File: src/test/java/uk/gov/justice/digital/hmpps/oauth2server/nomis/model/StaffTest.kt
package uk.gov.justice.digital.hmpps.oauth2server.nomis.model

import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

class StaffTest {
  @Test
  fun firstName() {
    val staff = Staff()
    staff.firstName = "boB"
    assertThat(staff.firstName).isEqualTo("Bob")
  }

  @Test
  fun name() {
    val staff = Staff()
    staff.firstName = "boB"
    staff.lastName = "SMITH"
    assertThat(staff.name).isEqualTo("Bob Smith")
  }
}
